MediaBrowser.Model/Providers/RemoteSearchResult.cs

```diff
@@ -8,39 +9,48 @@ namespace MediaBrowser.Model.Providers
 {
     public class RemoteSearchResult : IHasProviderIds
     {
+        public RemoteSearchResult()
+        {
+            ProviderIds = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            Artists = Array.Empty<RemoteSearchResult>();
```

Comment:
what would happen if someone executes
```cs
var foo = RemoteSearchResult();
foo.Artists.Add(bar);
```
?
Note that for previous code that would probably work...

Reply:
It wouldn't it's an array, no `Add` method